src/views/Manual/Manual.scss

```diff
@@ -25,20 +24,29 @@
 
 		&__pager {
 			float: right;
```

Comment:
:| Again,  I know you didn't do this but could we convert this to flex?  Everything else is flex goodness.